js/forms/settersForms.js

```diff
@@ -0,0 +1,16 @@
+const setDisabledField = (field) => field.setAttribute('disabled', '');
+const setActiveField = (field) => field.removeAttribute('disabled');
+
+const setDisabledElements = (fields) => fields.forEach((field) => setDisabledField(field));
```

Comment:
Если есть колбек вида (arg) => func(arg), то можно сократить до передачи функции вместо колбека, чтобы не создавать лишнюю функцию
```suggestion
const setDisabledElements = (fields) => fields.forEach(setDisabledField);
```

Reply:
постараюсь запомнить, что можно ещё и так сокращать, поправил, спасибо :)